### PR TITLE
Add option to render the title independently of the text

### PR DESCRIPTION
**Pull request by mbaechtold**
Until now the title was only rendered when the pane had some text.

### CODE REVIEW

File: ftw/slider/browser/slider.pt [hunk 1]
```diff
@@ -19,10 +19,11 @@
                    tal:content="structure python:images.tag(alt='', title='')"
                    />
             <div class="sliderText"
-                 tal:condition="pane/text">
-              <p tal:condition="pane/title" tal:content="pane/title"
+                 tal:condition="python: pane.show_title or pane.text">
+              <p tal:condition="pane/show_title" tal:content="pane/title"
                   class="documentFirstHeading title"/>
-              <span tal:replace="structure pane/text/output" />
+              <span tal:condition="pane/text"
```
Comment (maethu) [inline ftw/slider/browser/slider.pt]: @mbaechtold Why did you change this condition `pane/text` may return a RichValue object with a empty text?

Comment (maethu) [inline ftw/slider/browser/slider.pt]: @mbaechtold We need the same feature in ftw.sliderblock.

Comment (mbaechtold) [inline ftw/slider/browser/slider.pt]: Yes @maethu, I will prepare the PR in `ftw.sliderblock` once this PR has been merged and a release has been done because I will need to access the new `show_title` field from `ftw.slider`. `ftw.sliderblock` will then need at least the release containing these changes.

Comment (mbaechtold) [inline ftw/slider/browser/slider.pt]: @maethu, if `pane.text` does not have any content, it is `None`, not a `RichTextValue`.

Comment (mbaechtold) [inline ftw/slider/browser/slider.pt]: This means that in the `setup.py` of `ftw.sliderblock` I will need to require something like:

```
install_requires=[
        'ftw.simplelayout [contenttypes] >= 1.12.0',
        'ftw.slider >= 3.1.0',  # <--- THIS
        'setuptools',
        'plone.dexterity',
        'plone.app.dexterity',
        'Plone',
    ],

```

**Comment by mbaechtold** [comment]
@maethu, how is the review going?

**Comment by mbaechtold** [comment]
@maethu, is there anything I can do to get this PR merged?
